### DISCUSSION TITLE
duplicate Instances logic

### DISCUSSION THREAD

**Discussion by seriousgong** [chaos-mesh/chaos-mesh#2638] [Q&A]
https://github.com/chaos-mesh/chaos-mesh/blob/5005201fdab583f46bef71c2e9e536f6766cc7b6/controllers/chaosimpl/networkchaos/trafficcontrol/impl.go#L63-L69

is it being used for some special cases？

**Comment by xlgao-zju** [comment]
I think this duplicated, and we can remove one of them.

**Reply by xlgao-zju** [reply]
this has been fixed by: https://github.com/chaos-mesh/chaos-mesh/pull/3378
